lib/vf/number.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Cppcheck - A tool for static C/C++ code analysis
+ * Copyright (C) 2007-2023 Cppcheck team.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef vfNumberH
+#define vfNumberH
+
+class TokenList;
+
+namespace ValueFlow
+{
+    void number(TokenList *tokenlist);
```

Comment:
This is only used be valueflow.cpp. I think it would be better to put all the declarations in valueflow.cpp or in a single header that valueflow.cpp includes.

Reply:
Everything in the `vf` folder is only used by `valueflow.cpp`. But having a convenience header with all of them in it would be fine I guess.

Reply:
done